media/CfgEditor/cfgeditor.js

```diff
@@ -0,0 +1,325 @@
+ */
+/*
+This code refers to
+https://github.com/microsoft/vscode-webview-ui-toolkit-samples/blob/b807107df40271e83ea6d36828357fdb10d71f12/default/hello-world/webview-ui/main.js
+*/
+
+// Get access to the VS Code API from within the webview context
+const vscode = acquireVsCodeApi();
+
+// Just like a regular webpage we need to wait for the webview
+// DOM to load before we can reference any of the HTML elements
+// or toolkit components
+window.addEventListener('load', main);
+
+// Main function that gets executed once the webview DOM loads
+function main() {
+  const inputPathCodicon = document.getElementById('inputPathCodicon');
+  inputPathCodicon.addEventListener('click', clickInputPathCodicon);
+
+  const importEnabled = document.getElementById('importEnabled');
+  const importDisabled = document.getElementById('importDisabled');
+  const importLink = document.getElementById('linkEnableImport');
+  importEnabled.addEventListener('click', disableImportStep);
+  importDisabled.addEventListener('click', enableImportStep);
+  importLink.addEventListener('click', enableImportStep);
+
+  const optimizeEnabled = document.getElementById('optimizeEnabled');
+  const optimizeDisabled = document.getElementById('optimizeDisabled');
+  const optimizeLink = document.getElementById('linkEnableOptimize');
+  optimizeEnabled.addEventListener('click', disableOptimizeStep);
+  optimizeDisabled.addEventListener('click', enableOptimizeStep);
+  optimizeLink.addEventListener('click', enableOptimizeStep);
+
+  const quantizeEnabled = document.getElementById('quantizeEnabled');
+  const quantizeDisabled = document.getElementById('quantizeDisabled');
+  const quantizeLink = document.getElementById('linkEnableQuantize');
+  quantizeEnabled.addEventListener('click', disableQuantizeStep);
+  quantizeDisabled.addEventListener('click', enableQuantizeStep);
+  quantizeLink.addEventListener('click', enableQuantizeStep);
+
+  const codegenEnabled = document.getElementById('codegenEnabled');
+  const codegenDisabled = document.getElementById('codegenDisabled');
+  const codegenLink = document.getElementById('linkEnableCodegen');
+  codegenEnabled.addEventListener('click', disableCodegenStep);
+  codegenDisabled.addEventListener('click', enableCodegenStep);
+  codegenLink.addEventListener('click', enableCodegenStep);
+
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const profileLink = document.getElementById('linkEnableProfile');
+  profileEnabled.addEventListener('click', disableProfileStep);
+  profileDisabled.addEventListener('click', enableProfileStep);
+  profileLink.addEventListener('click', enableProfileStep);
+
+  const outputPathCodicon = document.getElementById('outputPathCodicon');
+  outputPathCodicon.addEventListener('click', clickOutputPathCodicon);
+
+  const inputModelSearchCodicon = document.getElementById('inputModelSearch')
```

Comment:
(check-format noticed this)

```suggestion
  const inputModelSearchCodicon = document.getElementById('inputModelSearch');
```

By the way, is semicolon a option to end of code line in typescript compiler?

Reply:
In javascript/typescript, semicolon is not mandatory!
Although there are many arguments about using semicolon, our repository is using semicolon at the end of line :)